corehq/apps/app_manager/management/commands/add_form_translation_fields.py

```diff
@@ -0,0 +1,53 @@
+from corehq.apps.app_manager.management.commands.helpers import AppMigrationCommandBase
+from corehq.apps.app_manager.models import Application
+
+from dimagi.utils.chunked import chunked
+from corehq.util.log import with_progress_bar
+
+
+class ResourceNotFound(object):
+    pass
+
+
+class Command(AppMigrationCommandBase):
+    help = """
+    Populate new 'submit_label' and 'submit_notification_label' in the FormBase class with language translations
+    """
+
+    def get_apps(self):
+        return Application.objects.all()
+
+    def get_apps_to_migrate(self):
+        apps_to_migrate = []
+        apps = self.get_apps()
+        for app in chunked(with_progress_bar(apps), 100):
+            if self._app_need_to_be_migrated(app.build_spec.version):
+                apps_to_migrate.append(app)
+        return apps_to_migrate
+
+    def _app_need_to_be_migrated(self, app_version):
+        major, minor, patch = [int(x) for x in app_version.split('.')]
+        return major >= 2 and minor >= 53
+
+    def migrate_form(self, form, translations):
+        submit_label_dict = {}
+        form_submission_notification_label_dict = {}
+        for lang in translations.items():
+            submit_label_dict[lang] = "Submit"
+            form_submission_notification_label_dict[lang] = "${0} successfully saved!"
```

Comment:
Is it necessary to populate these on all apps?  I think if you set this as default on the model, it'd work about the same (though maybe not for all langs).  Though actually I wonder if it wouldn't be better overall to leave these fields blank unless configured by the user, just using default text if not available, as you did already in `_get_success_message`.  Nice think about leaving them blank (besides skipping the migration) is that we can easily tell the difference between user-provided text and default text (and provide our own translations if the user didn't, for example).

Reply:
@esoergel ah I see so in that case it would not require any migration, correct?

Reply:
yeah that's right

Reply:
cool that sounds like the way to go-- I'll make those changes